core/domain/suggestion_registry.py

```diff
@@ -607,7 +610,7 @@ def accept(self, unused_commit_message):
                 'The skill with the given id doesn\'t exist.')
         question_services.create_new_question_skill_link(
             self.author_id, question_dict['id'], self.change.skill_id,
-            constants.DEFAULT_SKILL_DIFFICULTY)
+            self.skill_difficulty)
```

Comment:
Add a backend integration test to assure the skill_difficulty is set as expected when a review makes https request. (Integration test should be a part of the controller maybe?)

Reply:
Added skill_difficulty to controllers.suggestion_test.py.